scripts/promos.js

```diff
@@ -369,7 +372,9 @@ export async function renderSavedPromos() {
 
         // Sync only the balance of the code (not full data)
         cCode.getUTXOs(false);
-        const nBal = (await cCode.getBalance(true)) / COIN;
+
+        let nBal = ((await cCode.getBalance(true)) - PROMO_FEE) / COIN;
+        nBal = nBal < 0 ? 0 : nBal;
```

Comment:
Quick suggestion: this can be shortened down to...
```suggestion
        const nBal = Math.max((await cCode.getBalance(true)) - PROMO_FEE, 0) / COIN;
```

You'd need to run `npm run prettier` afterwards since this is longer than 80 chars wide (84).

Reply:
Unrelated to the PR, but why we have a constant `PROMO_FEE`?

Fee should be handled internally by the `TransactionBuilder`